src/kubernetes/software/cluster_autoscaler.cr

```diff
@@ -56,10 +56,37 @@ class Kubernetes::Software::ClusterAutoscaler
     end
   end
 
+  private def autoscaler_config_args
+    args = [] of String
+    
+    # Collect configuration from all autoscaling pools (use first value found)
+    autoscaling_configs = autoscaling_worker_node_pools.map(&.autoscaling.not_nil!).compact
+    
+    return args if autoscaling_configs.empty?
+    
+    # Use configuration from the first autoscaling pool
+    config = autoscaling_configs.first
```

Comment:
Sorry, maybe I wasn't clear. Right now you’re putting these settings on the first autoscaled node pool. But the autoscaling settings hit every autoscaled node pool, so I’d rather have a root-level block called `autoscaler` for them instead of pulling them off a node pool. Does that make sense?